ldk/javascript/src/webpack/generate-banner.ts

```diff
@@ -1,16 +1,33 @@
-export function generateMetadata(): string {
+import { LdkSettings } from './ldk-settings';
+
+export function generateMetadata(ldkSettings: LdkSettings): string {
+  if(Object.keys(ldkSettings.ldk).length === 0) {
+    throw new Error("Please provide LDK settings in your Loop package.json. See README for more information.")
+  }
   const json = JSON.stringify({
     ldkVersion: '0.1.0',
+    permissions: {
+      clipboard: ldkSettings.ldk.permissions.clipboard || undefined,
```

Comment:
The specification we discussed was that a missing key would be treated as a no-access request... and `JSON.stringify({value: undefined})` returns `"{}"`.

Maybe we should capture that expectation in a comment so we memorialize the expected behaviour.

Actually, maybe we should capture that in a test.

Reply:
`JSON.stringify` will omit properties with `undefined` values, so it's yielding the behavior we expect here. Maybe not the most straightforward way to do it, but we can always iterate on this.

Capturing this behavior in a test is a good idea.